### PR TITLE
Add article on contributing to GeeksBlabla community

### PR DESCRIPTION
**Pull request by kaizendae**
Introduce a comprehensive guide detailing various ways individuals can contribute to the GeeksBlabla community, emphasizing the impact of each contribution and encouraging participation from all members.

### CODE REVIEW

File: articles/10-ways-to-contribute-to-geeksblabla.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+## 7. Organize Meetups / Attend Meetups
+
+- **Impact:** ⭐⭐⭐⭐⭐
+- **Difficulty:** ⭐⭐⭐
+- **Time Investment:** ⭐⭐⭐
+
+If you’re a student or part of a university/school, it’s the perfect time to organize or attend meetups.
+
+Hosting meetups brings like-minded people together, creates opportunities to network, and fosters collaboration. Invite speakers, host discussions, or simply connect with others to share knowledge and build a stronger local tech network.
```
Comment (yjose) [inline articles/10-ways-to-contribute-to-geeksblabla.md:87]: I would like to mention that if you need more visibility, you can reach out to GeeksBlabla to share your meetup in our social media platform

File: articles/10-ways-to-contribute-to-geeksblabla.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## 1. Provide Feedback
+
+- **Impact:** ⭐⭐⭐
+- **Difficulty:** ⭐
+- **Time Investment:** ⭐
+
+Constructive feedback helps us improve. Whether it’s about our content, events, or initiatives, your input is invaluable in shaping the future of GeeksBlabla.
+
+Taking a few minutes to share your thoughts can lead to big improvements and better experiences for everyone involved.
```
Comment (yjose) [inline articles/10-ways-to-contribute-to-geeksblabla.md:23]: We can include the ability for people to suggest new concepts that we can work on.I know it's somewhat included here as part of the feedback, but I would love to make it explicit. The most useful aspect is that people are suggesting new concepts and projects for the community to work on.

File: articles/10-ways-to-contribute-to-geeksblabla.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+- **Difficulty:** ⭐
+- **Time Investment:** ⭐
+
+One of the best ways to contribute to the GeeksBlabla podcast is by suggesting topics and guests. This helps with one of our most challenging tasks, since we rely on a broad network of connections to find great guests.
+
+You might know ideal guests or niche topics that the core team hasn't discovered yet, helping us fill blind spots we wouldn't otherwise see. Plus, you'll benefit from other members' suggestions, discovering new topics and voices.
+
+Check our [planning here](#) to suggest an episode or a guest for an episode.
+
+## 5. Be a Guest on an Episode
```
Comment (yjose) [inline articles/10-ways-to-contribute-to-geeksblabla.md:57]: I feel that 4 and 5 are kind of the same.

Comment (kaizendae) [reply articles/10-ways-to-contribute-to-geeksblabla.md:61]: can you check it now

**Pull request merged**